test/cli/test_cli_functions.sh

```diff
@@ -284,6 +284,23 @@ function obc_cycle {
     echo_time "✅  obc cycle is done"
 }
 
+function crd_cycle {  
+    local crd 
+    echo_time "Starting the crd cycle"
+    test_noobaa timeout crd create
```

Comment:
running `noobaa crd create` when all crd exist will result in:

```
INFO[0000] ✅ Already Exists: CustomResourceDefinition "noobaas.noobaa.io"
INFO[0000] ✅ Already Exists: CustomResourceDefinition "backingstores.noobaa.io"
INFO[0000] ✅ Already Exists: CustomResourceDefinition "bucketclasses.noobaa.io"
INFO[0000] ✅ Already Exists: CustomResourceDefinition "objectbucketclaims.objectbucket.io"
INFO[0000] ✅ Already Exists: CustomResourceDefinition "objectbuckets.objectbucket.io"
```

in order to check that we need delete then create and check that it exists, this flow will test the delete command and the create command